sql-plugin/src/main/spark311/scala/org/apache/spark/sql/rapids/shims/spark311/RapidsShuffleInternalManager.scala

```diff
@@ -21,15 +21,7 @@ package org.apache.spark.sql.rapids.shims.spark311
 
 import org.apache.spark.SparkConf
 import org.apache.spark.shuffle._
-import org.apache.spark.sql.rapids.{ProxyRapidsShuffleInternalManagerBase, RapidsShuffleInternalManagerBase}
-
-/**
- * A shuffle manager optimized for the RAPIDS Plugin For Apache Spark.
- * @note This is an internal class to obtain access to the private
- *       `ShuffleManager` and `SortShuffleManager` classes.
- */
-class RapidsShuffleInternalManager(conf: SparkConf, isDriver: Boolean)
```

Comment:
With this removal, do we want to keep this updated as well https://github.com/NVIDIA/spark-rapids/blob/5b84728671c069f9c9755b8da259a55ef51de483/sql-plugin/src/main/scala/com/nvidia/spark/rapids/ShimLoader.scala#L133?

Reply:
Not unused but not inherently useful either, I'll keep working on the PR.